docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx

```diff
@@ -1,8 +1,10 @@
 ---
-title: "End to End Analysis Example"
+title: "Creating Visualizations Tutorial"
```

Comment:
nit: I would drop `Tutorial`

Maybe in future we add a Tutorials section this would go under

Reply:
updated in c560544b37fe5785fb8eac095e6156ecc457b4e0